packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/InputAwareWebView.java

```diff
@@ -124,18 +104,52 @@ public void run() {
   @Override
   public void clearFocus() {
     super.clearFocus();
+    resetInputConnection();
+  }
+
+  /**
+   * Ensure that input creation happens back on {@link #containerView}'s thread.
```

Comment:
Maybe: `Ensure that input connection creation happens back on {@link #containerView}`.

Reply:
Done.